Makefile

```diff
@@ -296,6 +296,9 @@ define CPU_PYTORCH_TAGS
 endef
 endif
 
+export DEFAULT_CPU_IMAGE=$(DOCKERHUB_REGISTRY)/$(CPU_PYTORCH_ENVIRONMENT_NAME):$(SHORT_GIT_HASH)
+export DEFAULT_CUDA_IMAGE=$(DOCKERHUB_REGISTRY)/$(CPU_PYTORCH_ENVIRONMENT_NAME):$(SHORT_GIT_HASH)
```

Comment:
```suggestion
export DEFAULT_CUDA_IMAGE=$(DOCKERHUB_REGISTRY)/$(CUDA_PYTORCH_ENVIRONMENT_NAME):$(SHORT_GIT_HASH)
```
?

Reply:
Disregard, this was not even close to the correct thing I should be exporting...